technical-reference/reference-technical-articles/xvs-bridge.md

```diff
@@ -212,3 +216,23 @@ In addition to the core functionality, the XVS Cross-chain Bridge includes addit
 - The transfer amount exceeds the single or daily receiving transaction limit.
 - The recipient is blacklisted by the XVS token.
 - The minting cap on the destination bridge is exceeded.
+
+### Retry Mechanism for Failed Transactions
```

Comment:
Generally, I would add:

* Mention and link (to the `main` branch version) to the contract where the retryMessage function is available
* Mention and link (to the `main` branch version) to the contract where the `MessageFailed` event is emitted